ethpm_cli/commands/install.py

```diff
@@ -119,6 +129,93 @@ def get_package_aliases(package_name: str, config: Config) -> Iterable[Tuple[str
                 yield alias
 
 
+def resolve_pkg_name_or_alias(pkg_name: str, config: Config) -> InstalledPackage:
+    lockfile_path = config.ethpm_dir / "ethpm.lock"
+    lockfile = json.loads(lockfile_path.read_text())
+    return InstalledPackage(**lockfile[pkg_name])
```

Comment:
You *might* check what mypy says about `reveal_type(lockfile)`.  This is more of a nitpick to try and get more confidence in the types for instances of `InstalledPackage` but you *might* consider something like this as I've found that `mypy` doesn't necesarily get along well with `**kwargs` style code.

```python
Lockfile = TypedDict(...) define the types for a lockfile dictionary somewhere.

class InstalledPackage:
    @classmethod
    def from_lockfile(cls, lockfile: Lockfile) -> 'InstalledPackage':
        return cls(
            arg1=lockfile['arg1'],  # these will be strongly typed due to the `TypedDict`
            ...
        )
```

This may not actually be a very big improvement and *maybe* mypy is doing better with this stuff so feel free to disregard this entirely or do some checking with `reveal_type(...)` to figure out if there's even a problem with the types.